### PR TITLE
Feat/toggledrawer => added toggle feature for drawer, created separate base components for quote types.

### PR DESCRIPTION
**Pull request by MAfzalKhan1997**

### CODE REVIEW

File: pages/dashboard/index.tsx [hunk 1]
```diff
@@ -99,7 +99,7 @@ const MainPage = () => {
   return (
     <>
       <NextSeo
-        title='Title for this page'
+        title='Dashbaord'
```
Comment (hussainanjar) [inline pages/dashboard/index.tsx:102]: We need arabic as well side by side

Comment (MAfzalKhan1997) [reply pages/dashboard/index.tsx:102]: Noted! will implement in next PR

**Review by umimehar — COMMENTED** [review]
Please dont put everything in base components. base components are low level html components. like button, input, image or video tag or any specific image like logo.

**Comment by MAfzalKhan1997** [comment]
> Please dont put everything in base components. base components are low level html components. like button, input, image or video tag or any specific image like logo.

Thanks for guidance! will restructure it in next PR

**Pull request merged**